### ISSUE TITLE
Add gamelist scrolling sound

### ISSUE THREAD

**Issue by sergioad** [boluge/es_theme_recalbox_for_retropie#1]
Hi friend, I love Recalbox's theme and I would love to use it on my PC but... Could you add a scroll sound? you could ask to the creators of the Carbon theme and ask them if you could use their sounds (the modern ones), change the theme's name to simple modern, Simple flat or simple batocera and add a credits file refering to the Recalbox and Carbon theme

**Comment by sergioad** [comment]
Then, you cfould add this theme to the ES theme downloader

**Comment by sdeluce** [comment]
You know how to add a theme to the ES Theme downloader ?

**Comment by sergioad** [comment]
yes, there is a github repo where you need to fill a pull request for it, ask meelu in the forums for help
